libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -101,6 +103,7 @@ class NavigationView @JvmOverloads constructor(
                     .accessToken(accessToken)
                     .build()
             )
+            SharedApp.setup(context.applicationContext as Application)
```

Comment:
_**Note:** As discussed we should not do this in the Drop-In UI. The way we discussed is to enforce the users to invoke `SharedApp.setUp()` in their `Application` class just as Android-Auto does. A follow up PR will solve that problem and introduce that mechanism._